web_src/css/repo.css

```diff
@@ -3350,3 +3350,24 @@ tbody.commit-list {
   font-size: 18px;
   margin-left: 4px;
 }
+
+.commit-list-summary {
+  max-width: max(40vw, 200px);
+}
+
+.commit-selector-menu {
+  max-height: max(45vh, 200px);
+  overflow-y: auto;
+}
+
+.ui.dropdown .menu.commit-selector-menu > .item.vertical {
+  padding: 7px 14px !important;
+}
+
+.ui.dropdown .menu.commit-selector-menu > .item {
+  line-height: 1.4;
+}
+
+.ui.dropdown .menu.commit-selector-menu > .divider {
+  margin: 0;
+}
```

Comment:
If we do not need the "margin" for a divider, then maybe there is no need to use "divider", add a "gt-border-top" to the items?

Reply:
yes - removed it and added the border-top